source/HL_pinmux.c

```diff
@@ -269,7 +269,7 @@ void muxInit(void){
 	
 	pinMuxReg->PINMUX[17] = PINMUX_BALL_R8_EMIF_DATA_15 | PINMUX_BALL_R9_GIOA_5 | PINMUX_BALL_R10_GIOA_6 | PINMUX_BALL_R11_GIOA_7;
 	
-	pinMuxReg->PINMUX[18] = PINMUX_BALL_B15_GIOA_2 | PINMUX_BALL_B8_GIOB_0 | PINMUX_BALL_B16_GIOB_1 | PINMUX_BALL_B9_GIOB_2;
+	pinMuxReg->PINMUX[18] = PINMUX_BALL_B15_GIOA_2 | PINMUX_BALL_B8_GIOB_0 | PINMUX_BALL_B16_GIOB_1 | PINMUX_BALL_B9_FRAYTXEN2;

```

Comment:
Why are we using FRAY pins?

Reply:
We're not. It's just an option for this specific pin. Im not sure how this code works exactly but you can pull up halcogen and see thay no flexray is enabled ot pinmuxd